.pre-commit-config.yaml

```diff
@@ -1,10 +1,18 @@
 repos:
-- repo: https://github.com/ambv/black
-  rev: stable
-  hooks:
-  - id: black
-    language_version: python3
-- repo: https://github.com/pre-commit/mirrors-mypy
-  rev: ""
-  hooks:
-  - id: mypy
\ No newline at end of file
+  - repo: "https://github.com/pycqa/isort"
+    rev: 5.7.0
+    hooks:
+      - id: isort
+        args:
+          - "--profile"
+          - black
+        name: isort
+  - repo: "https://github.com/ambv/black"
+    rev: stable
```

Comment:
[This](https://github.com/Dekalabs/cookiecutter-backend/commit/dbf588cb3ea17c7a552602ac62aee428edc4ce35) might be relevant here

Reply:
Done